apps/stable_diffusion/src/models/model_wrappers.py

```diff
@@ -502,6 +509,13 @@ def forward(self, latent, timestep, text_embedding, noise_level):
         unet = UnetModel(low_cpu_mem_usage=self.low_cpu_mem_usage)
         is_f16 = True if self.precision == "fp16" else False
         inputs = tuple(self.inputs["unet"])
+        if(sequence_length > 64):
```

Comment:
Should this be `if(sequence_length > arg.max_length` or something equivalent if args.maxlength in passed in? 

Same for other places where `64` is hardcoded.

That's not a rhetorical question, I don't actually know :)

Reply:
Oh, good point. I should probably just pass a bool there, something like `use_large` or `use_512`.